genai-perf/tests/test_json_exporter.py

```diff
@@ -229,137 +230,139 @@ def test_generate_json_custom_export(
-          },
-          "output_token_throughput": {
-              "unit": "tokens/sec",
-              "avg": 31
-          },
-          "output_token_throughput_per_request": {
-              "unit": "tokens/sec",
-              "avg": 41,
-              "p99": 42,
-              "p95": 43,
-              "p90": 44,
-              "p75": 45,
-              "p50": 46,
-              "p25": 47,
-              "max": 48,
-              "min": 49,
-              "std": 40
-          },
-          "output_sequence_length": {
-              "unit": "tokens",
-              "avg": 51,
-              "p99": 52,
-              "p95": 53,
-              "p90": 54,
-              "p75": 55,
-              "p50": 56,
-              "p25": 57,
-              "max": 58,
-              "min": 59,
-              "std": 50
-          },
-          "input_sequence_length": {
-              "unit": "tokens",
-              "avg": 61,
-              "p99": 62,
-              "p95": 63,
-              "p90": 64,
-              "p75": 65,
-              "p50": 66,
-              "p25": 67,
-              "max": 68,
-              "min": 69,
-              "std": 60
-          },
-        "input_config": {
-          "model": ["gpt2_vllm"],
-          "formatted_model_name": "gpt2_vllm",
-          "model_selection_strategy": "round_robin",
-          "backend": "vllm",
-          "batch_size": 1,
-          "endpoint": null,
-          "endpoint_type": null,
-          "service_kind": "triton",
-          "server_metrics_url": null,
-          "streaming": true,
-          "u": null,
-          "input_dataset": null,
-          "num_prompts": 100,
-          "output_tokens_mean": -1,
-          "output_tokens_mean_deterministic": false,
-          "output_tokens_stddev": 0,
-          "random_seed": 0,
-          "synthetic_input_tokens_mean": 550,
-          "synthetic_input_tokens_stddev": 0,
-          "image_width_mean": 100,
-          "image_width_stddev": 0,
-          "image_height_mean": 100,
-          "image_height_stddev": 0,
-          "image_format": null,
-          "concurrency": 1,
-          "measurement_interval": 10000,
-          "request_rate": null,
-          "stability_percentage": 999,
-          "generate_plots": false,
-          "profile_export_file": "artifacts/gpt2_vllm-triton-vllm-concurrency1/profile_export.json",
-          "artifact_dir": "artifacts/gpt2_vllm-triton-vllm-concurrency1",
-          "tokenizer": "hf-internal-testing/llama-tokenizer",
-          "verbose": false,
-          "goodput": null,
-          "subcommand": "profile",
-          "prompt_source": "synthetic",
-          "extra_inputs": {
-            "max_tokens": 256,
-            "ignore_eos": true
-          }
+        {
+            "llm_metrics": {
+                "request_throughput": {
+                "unit": "requests/sec",
+                "avg": "7"
+                },
+                "request_latency": {
+                "unit": "ms",
```

Comment:
I would not expect the formatting to change this much.
Are you running the same pre commit hook?